lesson_07/proto/proto.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+option go_package = "proto/pb";
+
+service Echo {
+    rpc SayHello(Message) returns (Message) {}
+    rpc Register(RegisterRequest) returns (RegisterResponse) {}
+    rpc List(Empty) returns (ListResponse) {}
```

Comment:
Did not get how to do it without this custom Empty message

Reply:
Checked examples from current project, it seems we need provide some request parameter, even it has no fields. Our project application uses the same pattern, provides empty parameter